### PR TITLE
disable menu items according to complevel

### PR DESCRIPTION
**Pull request by rfomin**

### CODE REVIEW

**Comment by rfomin** [comment]
Probably we should disable the "classic BFG" option, but I want to leave it just for fun :) I think it is obvious to everyone that it is not compatible with anything.

File: Source/m_menu.c [hunk 1]
```diff
@@ -6123,6 +6150,33 @@ void M_ResetSetupMenu(void)
   SetupMenu[0].status = (demo_version < 203) ? 0 : 1;
 }
 
+void M_ResetSetupMenuItems(void)
+{
+  static boolean toggle_boom = false;
```
Comment (fabiangreffrath) [inline Source/m_menu.c:6155]: Hm, I don't like the toggle solution and would prefer to set the flags explicitly depending on the currenr `demo_version` alone, not on the value of another variable that was set during the previous call.

**Comment by rfomin** [comment]
>  Do its code pointers work in lower complevels?

Yes, I didn't change it on purpose either :)

**Comment by fabiangreffrath** [comment]
> Yes, I didn't change it on purpose either :)

Whatever you like. 😉

File: Source/m_menu.c [hunk 2]
```diff
@@ -6120,7 +6146,39 @@ void M_ResetMenu(void)
 
 void M_ResetSetupMenu(void)
 {
-  SetupMenu[0].status = (demo_version < 203) ? 0 : 1;
+  SetupMenu[set_compat].status = (demo_version < 203) ? 0 : 1;
```
Comment (fabiangreffrath) [inline Source/m_menu.c:6149]: Sorry, but could you please integrate it into the other checks?

Comment (rfomin) [reply Source/m_menu.c]: Fixed in https://github.com/fabiangreffrath/woof/pull/195/commits/facc1ac00d539112f617272ef956122c7d1635f0

**Comment by fabiangreffrath** [comment]
Disable all the things! 💪

**Pull request merged**